frontend/packages/knative-plugin/src/models.ts

```diff
@@ -0,0 +1,66 @@
+import { K8sKind } from '@console/internal/module/k8s';
+
+export const ConfigurationModel: K8sKind = {
+  apiGroup: 'serving.knative.dev',
+  apiVersion: 'v1alpha1',
+  kind: 'Configuration',
+  plural: 'configurations',
+  label: 'Configuration',
+  labelPlural: 'Configurations',
+  id: 'configuration',
+  abbr: 'KC',
+  namespaced: true,
+  crd: true,
+};
+
+export const KnativeServingModel: K8sKind = {
+  apiGroup: 'serving.knative.dev',
+  apiVersion: 'v1alpha1',
+  kind: 'KnativeServing',
+  label: 'Knative Serving',
+  labelPlural: 'knativeservings',
+  plural: 'knativeservings',
+  id: 'knativeserving',
+  abbr: 'KSV',
```

Comment:
am not sure how `abbr` works but when i try `store.getState().k8s.toJS().RESOURCES.models['serving.knative.dev~v1alpha1~KnativeServing']` in browser console registered model has `KS` as abbr

Reply:
fixed